modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
+
+pipe="$HOME/tmp/print-notify"
+icon=/opt/share/xsession/images/ocf-color-256.png
+
+[[ -p "$pipe" ]] && rm "$pipe"
+mkfifo $pipe
+
+sudo -u ocfbroker /opt/share/puppet/print-notify-handler > "$pipe" &
+
+# listener
+while true
+do
```

Comment:
Interesting you put the `then` on the same line but don't do the same for `do`...